app/forms/idv/api_image_upload_form.rb

```diff
@@ -111,8 +111,11 @@ def validate_pii_from_doc(client_response)
         attention_with_barcode: client_response.attention_with_barcode?,
       ).submit
       response.extra.merge!(extra_attributes)
+      doc_type_and_issuing_country = doc_type_and_issuing_country(response.pii_from_doc)
```

Comment:
@night-jellyfish  Not sure best place to handle the doc type and issue country is here. On line 109, the DocPiiForm is one place to centralize the information collection, and deal with filed name transformation, in which case issue on line 127, wont be an issue anymore(@zachmargolis)

Reply:
@night-jellyfish , giving another look at the card, seems we wanna log both sides info if possible, so I would say it's acceptable to process it outside of the pii_from_doc where it's more like a unified confirmed source.

Reply:
Ok, thank you! So just to clarify: no change is needed here?